packages/beacon-node/test/perf/api/impl/validator/attester.test.ts

```diff
@@ -1,5 +1,6 @@
 import {itBench} from "@dapplion/benchmark";
 import {PointFormat} from "@chainsafe/bls/types";
+// eslint-disable-next-line import/no-relative-packages
 import {generatePerfTestCachedStatePhase0, numValidators} from "../../../../../../state-transition/test/perf/util.js";
```

Comment:
I feel like we should get rid of those cross-package relative imports. Those functions could be moved to @lodestar/test-utils and then imported from there.

Reply:
Yes we can do that and `@lodestar/test-utils` will be helpful in this context.